api/src/pcapi/core/offerers/models.py

```diff
@@ -575,6 +575,18 @@ def current_reimbursement_point_id(self) -> int | None:
             .scalar()
         )
 
+    @current_reimbursement_point_id.expression  # type: ignore [no-redef]
+    def current_reimbursement_point_id(cls) -> str | None:  # pylint: disable=no-self-argument # type: ignore[no-redef]
```

Comment:
Je ne vois cette expression utilisée dans aucun filtre, donc je soupçonne qu'elle soit inutile.

Si jamais on garde, j'imagine que `str` n'est pas volontaire : 
```suggestion
    def current_reimbursement_point_id(cls) -> int | None:  # pylint: disable=no-self-argument # type: ignore[no-redef]
```